__tests__/service/movie/sorter.test.js

```diff
@@ -8,6 +8,12 @@ describe('Movie sorter service', () => {
         const m1 = new Movie(Movie.TYPE_MOVIE, 'Fight Club', 'pic', 50, 50);
         const m2 = new Movie(Movie.TYPE_MOVIE, 'Snatch', 'pic', 50, 100);
         const m3 = new Movie(Movie.TYPE_TV_SHOW, 'Snatch', 'pic', 50, 100);
+        const m3 = Movie.tvShow({
```

Comment:
here I'd use a static constructor instead of reaching for a constant in the movie model (which I should've renamed Show). I prefer functions as constants.
I would also use an object as the fields are name and more self-documenting.
It allows you to just not pass a field if it is optional.